projects/admin/src/app/menu/menu.component.ts

```diff
@@ -40,6 +40,7 @@ export class MenuComponent implements OnInit {
 
   maxLengthSuggestion = 100;
 
+
```

Comment:
I think this blank line should be removed

Reply:
blank line removed.